hieradata/nodes/dev-anthrax.yaml

```diff
@@ -0,0 +1 @@
+ocf::packages::postfix::manage: false
```

Comment:
Things are a lot nicer now, that does look like a bit of a mess. `tagged` works fine, but I see your point that it's brittle, so it would be good to use roles instead, since we're most of the way there. I'll look into being able to have multiple roles somehow, it could be useful.